internal/bgp/frr/frr.go

```diff
@@ -282,19 +282,24 @@ func (sm *sessionManager) createConfig() (*frrConfig, error) {
 			family := ipfamily.ForAddress(adv.Prefix.IP)
 
 			communities := make([]string, 0)
+			largeCommunities := make([]string, 0)
 
 			// Convert community 32bits value to : format
 			for _, c := range adv.Communities {
-				community := metallbconfig.CommunityToString(c)
-				communities = append(communities, community)
+				if c.IsLarge() {
+					largeCommunities = append(largeCommunities, c.String())
```

Comment:
mind continue-ing and appending otherwise (instead of the else)?

Reply:
done